moveit_kinematics/cached_ik_kinematics_plugin/README.md

```diff
@@ -54,7 +54,7 @@ Below is a complete list of all arguments:
 
 ## Advanced Usage: Creating Wrappers for Other IK Solvers
 
-The Cached IK Kinematics Plugin is implemented as a wrapper around classed derived from the [`kinematics::KinematicsBase` abstract base class](http://docs.ros.org/latest-lts/api/moveit_core/html/classkinematics_1_1KinematicsBase.html). Wrappers for the `kdl_kinematics_plugin::KDLKinematicsPlugin` and `srv_kinematics_plugin::SrvKinematicsPlugin` classes are already included in the plugin. For any other solver, you can create a new kinematics plugin. The C++ code for doing so is extremely simple; here is the code to create a wrapper for the KDL solver:
+The Cached IK Kinematics Plugin is implemented as a wrapper around classed derived from the `kinematics::KinematicsBase` [abstract base class](http://docs.ros.org/en/noetic/api/moveit_core/html/cpp/classkinematics_1_1KinematicsBase.html). Wrappers for the `kdl_kinematics_plugin::KDLKinematicsPlugin` and `srv_kinematics_plugin::SrvKinematicsPlugin` classes are already included in the plugin. For any other solver, you can create a new kinematics plugin. The C++ code for doing so is extremely simple; here is the code to create a wrapper for the KDL solver:
```

Comment:
Should we just use `http://docs.ros.org/en/latest/...` instead?

Reply:
Updated, thanks